odl/contrib/datasets/images/cambridge.py

```diff
@@ -0,0 +1,145 @@
+# Copyright 2014-2017 The ODL contributors
+#
+# This file is part of ODL.
+#
+# This Source Code Form is subject to the terms of the Mozilla Public License,
+# v. 2.0. If a copy of the MPL was not distributed with this file, You can
+# obtain one at https://mozilla.org/MPL/2.0/.
+
+"""Images provided by the University of Cambridge."""
+
+import numpy as np
+from odl.contrib.datasets.util import get_data
+from scipy.misc import imresize
+
+__all__ = ('brain_phantom', 'resolution_phantom', 'building', 'rings',
+           'blurring_kernel')
+
+
+DATA_SUBSET = 'images_cambridge'
+URL_CAM = 'http://store.maths.cam.ac.uk/DAMTP/me404/data_sets/'
+
+
+def convert(image, shape, gray=None, dtype='float64', normalize='max'):
+    """Convert image to standardized format.
+
+    Several properties of the input image may be changed including the shape,
+    data type and maximal value of the image. In addition, this function may
+    convert the image into an ODL object and/or a gray scale image.
+    """
+
+    image.astype(dtype)
+
+    if gray is not None and gray:
+        image = np.mean(image, 2)
```

Comment:
I don't think `mean` is the best way to convert color images to grayscale. I'd suggest you try to compare with
```python
image[..., 0] *= 0.2126
image[..., 1] *= 0.7152
image[..., 2] *= 0.0722
image = np.sum(image, axis=2)
```
see [here](https://en.wikipedia.org/wiki/Grayscale#Colorimetric_.28perceptual_luminance-preserving.29_conversion_to_grayscale).

Reply:
Sure, I can change this. I don't think it is very important for these images though.